lib/solana_rpc_ruby/redis_client.rb

```diff
@@ -0,0 +1,45 @@
+require 'redis'
+
+module SolanaRpcRuby
+  class RedisClient
+    DEFAULT_EXPIRATION_TIME = 10 # sec
+
+    def initialize(
+      host: 'localhost', 
+      expire: true, 
+      expiration_time: DEFAULT_EXPIRATION_TIME,
+      set_name: 'solana_sorted_set'
+    )
+      @redis = Redis.new(host: host)
+      @expire = expire
+      @expiration_time = expiration_time
+      @set_name = set_name
+      @score = get_highest_score.first[1]
```

Comment:
Use safe operator here. It caused some errors on starting the 'demo_ws' and 'demo_redis' scripts.

Reply:
Redis is no longer used now.